packages/mcp-provider-dx-core/package.json

```diff
@@ -133,9 +120,12 @@
         "src/**/*.ts",
         "**/tsconfig.json",
         ".mocha*",
-        "!*.nut.ts",
         ".nycrc"
       ],
+      "dependencies": [
+        "test:compile",
+        "lint"
+      ],
```

Comment:
so now when you run `yarn lint` inside `packages/mcp-dx-core` it will
1. try to compile TS test files to catch type errors
2. lint 

see wireit's dependencies:
https://github.com/google/wireit/?tab=readme-ov-file#dependencies